walk_through.ipynb

```diff
@@ -1,103 +0,0 @@
-{
```

Comment:
is this file empty?

Reply:
This file is deleted, so I think that's why it shows as empty